packages/vue-client/.eslintrc.cjs

```diff
@@ -16,6 +16,6 @@ module.exports = {
     },
   ],
   rules: {
-    "prettier/prettier": "error",
+    "prettier/prettier": ["error", { endOfLine: "auto" }],
```

Comment:
I know you had mentioned adding a config might help VSCode ignore your global config. Does this do that? Or we could move this into its own file if necessary.